src/styles/components/checkbox.scss

```diff
@@ -0,0 +1,3 @@
+.overtime-checkbox {
+  display: flex !important;
```

Comment:
We already have [a class which adds `display: flex`](https://github.com/LBHackney-IT/repairs-hub-frontend/blob/33d0517d26232c8d7332877094d52154e74d96eb/src/styles/layout.scss#L1). Can we use that instead of making a new class and CSS rule or is the `!important` definitely needed?